src/checkpoint/serializers/footprinter.h

```diff
@@ -61,7 +61,7 @@ struct Footprinter : Serializer {
   }
 
   template<typename T>
```

Comment:
While we're working here, could you also add a simple `void addBytes()` method that can be used for containers in the runtime that might keep track of the footprint of objects they're holding, but which can't be traversed directly (e.g. the message queues)

Reply:
Body would literally be `num_bytes_ += s;`